checklist_plus/text_generation/masked_lm.py

```
@@ -1,335 +1,287 @@
+                text += ' '
+        text += pieces[-1]
+        if pieces[-1] == '':
+            text = text.rstrip()
+        return self.unmask(text, beam_size=beam_size, candidates=candidates)
+
+    def replace_word(self, text, word,  threshold=5, beam_size=100, candidates=None):
+        masked = re.sub(r'\b%s\b' % re.escape(word), self.tokenizer.mask_token, text)
+        if masked == text:
+            return []
+        if candidates is not None:
+            candidates = [word] + candidates
```

Ruff (0.12.2)

Consider `[word, *candidates]` instead of concatenation

Replace with `[word, *candidates]`

(RUF005)